packages/d3fc-webgl/src/series/glArea.js

```diff
@@ -10,22 +11,22 @@ export default () => {
     let yScale = glScaleBase();
     let decorate = () => {};
 
-    const xValueAttribute = projectedAttributeBuilder().value(
+    const xValueAttribute = elementConstantAttributeBuilder().value(
         (data, element) => data[Math.min(element + 1, data.length - 1)]
     );
-    const xPreviousValueAttribute = projectedAttributeBuilder().value(
+    const xPreviousValueAttribute = elementConstantAttributeBuilder().value(
```

Comment:
I don't think we need to explicitly set `value` as it's the same as the default. Please could you check for this in the other changes?